src/index.js

```diff
@@ -43,47 +43,130 @@ const patch = snabbdom.init([
 	const poll = JSON.parse(await archive.readFile('/poll.json'))
 	const votes = []
 	const regex = new RegExp('^[a-z0-9]{8}\.json$', 'i')
-	for (let file of await archive.readdir('/votes')) {
-		const isJson = regex.test(file)
-		if (isJson) {
-			try {
-				const vote = JSON.parse(await archive.readFile('/votes/' + file))
-				if (isValidVote(vote)) {
-					votes.push(vote)
+	const isJson = (fileName) => regex.test(fileName)
+	const localVotes = await archive.readdir('/votes')
+	await Promise.all(
+		localVotes
+		.map(async (file) => {
+			if (isJson(file)) {
+				try {
+					const vote = JSON.parse(await archive.readFile('/votes/' + file))
+					if (isValidVote(vote)) {
+						votes.push(vote)
+					}
+				} catch(e) {
+					// TODO: user UI feedback
+					console.error(e)
```

Comment:
`throw` here, it will be logged to the console anyways. the `TODO` makes more sense at https://github.com/derhuerst/poodle-to-poodle/pull/4/files#diff-1fdf421c05c1140f6d71444ea2b27638R176